rules/Proj1002.md

```diff
@@ -0,0 +1,20 @@
+# Proj1002: Use Microsoft's analyzers
+[microsoft](https://microsoft.com) has implemented multiple generic
+Roslyn static code analysis rules both for C# and Visual Basic. These rules
+allow you to produce safe, reliable and maintainable code by helping you find
+and correct bugs, vulnerabilities and code smells in your codebase. It is
+strongly advised to enable this rules on all your projects.
+
+See: [learn.microsoft.com/dotnet/fundamentals/code-analysis/overview](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/overview)
+
+## Compliant
+``` XML
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
```

Comment:
For .NET 5 and later, this rule is enabled by default. Maybe you can mention that for earlier frameworks it needs to be enabled, but that it might be good to **explicitly** enable it for later frameworks as well.